inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/preprocess.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+
+#include "../base/layer_test_utils.hpp"
+#include "../../../../../ngraph_helpers/ngraph_functions/include/ngraph_functions/builders.hpp"
+#include "../../../../../ngraph_helpers/ngraph_functions/include/ngraph_functions/preprocess/preprocess_builders.hpp"
+#include "../../../../../ngraph_helpers/ngraph_functions/include/ngraph_functions/utils/ngraph_helpers.hpp"
```

Comment:
I think we just need to link `ngraph_helpers` with `shared_test_classes` library and don't use relative paths